Utils/azuretools-core/pom.xml

```diff
@@ -184,5 +184,10 @@
             <groupId>com.google.code.gson</groupId>
             <artifactId>gson</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>18.0</version>
```

Comment:
why we have version information here?

Reply:
Removed

---
In reply to: [154040202](https://github.com/Microsoft/azure-tools-for-java/pull/1279#discussion_r154040202) [](ancestors = 154040202)